mmv1/third_party/terraform/utils/provider_test.go.erb

```diff
@@ -110,175 +93,19 @@ var billingAccountEnvVars = []string{
 	"GOOGLE_BILLING_ACCOUNT",
 }
 
-var configs map[string]*Config
-
-// A source for a given VCR test with the value that seeded it
-type VcrSource struct {
-	seed int64
-	source rand.Source
-}
-
-var sources map[string]VcrSource
-
 // This is the billing account that will be modified to test billing-related functionality. It is
 // expected to have more permissions granted to the test user and support subaccounts.
 var masterBillingAccountEnvVars = []string{
 	"GOOGLE_MASTER_BILLING_ACCOUNT",
 }
 
-var configsLock = sync.RWMutex{}
-var sourcesLock = sync.RWMutex{}
-
 func init() {
 	configs = make(map[string]*Config)
 	sources = make(map[string]VcrSource)
 	testAccProvider = Provider()
-	<% if version == 'ga' -%>
-	TestAccProviders = map[string]*schema.Provider{
-		"google": testAccProvider,
-	}
-	<% else -%>
 	TestAccProviders = map[string]*schema.Provider{
```

Comment:
note: still needed for an upgrade test- all other users are gone (hence why deleting oics is safe)